scripts/validate-and-merge.sc

```diff
@@ -0,0 +1,67 @@
+import $ivy.`io.circe::circe-generic:0.14.3`
+import $ivy.`io.circe::circe-parser:0.14.3`
+import $ivy.`com.lihaoyi::os-lib:0.9.0`
+import $ivy.`com.lihaoyi::ammonite-ops:2.4.0`
+
+import ammonite.ops._
+import io.circe.jawn._
+import io.circe._
+import io.circe.syntax._
+import io.circe.generic.semiauto._
+
+import java.nio.channels.Channels
+
+import $file.Model
+
+@main
+def main(output: String): Unit = {
+
+  val dapps = (ls ! pwd / 'dapps)
+    .filter(_.isDir)
+    .map(_ / "dapp-allowlist.json")
+    .map { path =>
+      println(s"Validating ${path}.")
+      validate[Model.DappAllowList](path) match {
+        case Left(error) =>
+          println(s"Error: ${error.getMessage}")
```

Comment:
getMessage is often not super explicit with circe, please try to get backend-commons-circe (or duplicate code here)